main.lua

```diff
@@ -55,28 +55,29 @@ end
 local sqlite = require("sqlite3")
 local db = sqlite.open("db.sqlite")
 
-db:exec[[
+db:exec([[
 CREATE TABLE IF NOT EXISTS "members" (
   id TEXT PRIMARY KEY NOT NULL,
   points INTEGER NOT NULL DEFAULT 0
 )
-]]
+]])
 
-local stmt = db:prepare("INSERT INTO members (id, points) VALUES(?, ?) ON CONFLICT DO UPDATE SET points = points + ? RETURNING *")
+local stmt =
+  db:prepare("INSERT INTO members (id, points) VALUES(?, ?) ON CONFLICT DO UPDATE SET points = points + ? RETURNING *")
```

Comment:
We may want to raise `column_width` in `stylua.toml` from its default of 120 because this hurts me just to look at.

Reply:
> 120 is pretty aggressive as it is.

Aggressively short or...

> Can we do something like this?
> 
> ```
> local stmt = db:prepare(
>   "INSERT INTO members (id, points) VALUES(?, ?) " ..
>   "ON CONFLICT DO UPDATE SET points = points + ? RETURNING *")
> ```

Absolutely, was planning to do something like that in the refactor PR.

Reply:
Aggressively long. IMO you want to be confidently under half a widescreen monitor with some reasonable assumptions about font size and such.

I don't think there's a strict reason this PR needs to only include stylua's changes - the nice thing to believe about it is that stylua has no objections as of this commit. So feel free to break it up now if it's eating at you. :)

Reply:
> Aggressively long. IMO you want to be confidently under half a widescreen monitor with some reasonable assumptions about font size and such.

Funny enough I do get 240 columns at my regular terminal font size.
So 120 is exactly half of that.
 
> I don't think there's a strict reason this PR needs to only include stylua's changes - the nice thing to believe about it is that stylua has no objections as of this commit. So feel free to break it up now if it's eating at you. :)

As in, include the refactor commit(s) in this PR?

Reply:
Well I was gonna do a refactoring pass anyway in another PR.